sbt-twirl/src/main/scala/play/twirl/sbt/TemplateProblem.scala

```diff
@@ -87,7 +88,7 @@ object TemplateProblem {
     }
 
     def apply(source: Option[File], codec: Codec): TemplateMapping = {
-      val lines = source.toSeq flatMap { file => IO.readLines(file, codec.charSet) }
+      val lines = source.toSeq flatMap { file => TwirlIO.readFileAsString(file, codec).split('\n') }
```

Comment:
Actually no. This will not work. You can use different new line separators. This depends on Git, SVN, etc. client configuration, IDE or other editor configuration.
I personally always try to use UNIX new lines when possible to avoid problems. Often have mixed after checkout with UNIX ones and some edits with an editor where it's not configurable.